octoprint_mrbeam/software_update_information.py

```diff
@@ -19,72 +24,197 @@
     SW_UPDATE_TIER_DEV: "develop",
 }
 
-# add to the display name to modules that should be shown at the top of the list
-SORT_UP_PREFIX = " "
-
+SW_UPDATE_FILE = "update_info.json"
+SW_UPDATE_CLOUD_PATH = "https://mr-beam.org/beamos/config/sw-update-conf"
 
 _logger = mrb_logger("octoprint.plugins.mrbeam.software_update_information")
 
+# sw_update_config = dict()
+
 # Commented constants are kept in case we update more packages from the virtualenv
 # GLOBAL_PY_BIN = "/usr/bin/python2.7"
 # VENV_PY_BIN = sys.executable
 GLOBAL_PIP_BIN = "/usr/local/bin/pip"
 GLOBAL_PIP_COMMAND = (
     "sudo {}".format(GLOBAL_PIP_BIN) if os.path.isfile(GLOBAL_PIP_BIN) else None
 )
-# GLOBAL_PIP_COMMAND = "sudo {} -m pip".format(GLOBAL_PY_BIN) if os.path.isfile(GLOBAL_PY_BIN) else None #  --disable-pip-version-check
-# VENV_PIP_COMMAND = ("%s -m pip --disable-pip-version-check" % VENV_PY_BIN).split(' ') if os.path.isfile(VENV_PY_BIN) else None
+
 BEAMOS_LEGACY_DATE = date(2018, 1, 12)
 
+_softwareupdate_handler = None
```

Comment:
If you are going to use a singleton, then please explain (docstring) this variable

Reply:
ok